Gopkg.toml

```diff
@@ -62,8 +62,9 @@
   name = "github.com/juju/bundlechanges"
 
 [[constraint]]
-  revision = "04755bd60156009bd35788e17c06115c5532c233"
+  revision = "780d9315d7ca5097ab2b732e5b3d0f96c64b9fdd"
   name = "github.com/juju/description"
+  source = "github.com/hmlanigan/description.git"
```

Comment:
to be updated before merge, waiting on https://github.com/juju/description/pull/60